draft-ietf-mls-protocol.md

```diff
@@ -1314,10 +1314,9 @@ struct {
 } Extension;
 
 struct {
-    ProtocolVersion supported_version;
-    opaque client_init_key_id<0..255>;
+    ProtocolVersion version;
     CipherSuite cipher_suite;
-    HPKEPublicKey init_key;
+    HPKEPublicKey leaf_key;
```

Comment:
I prefer `init_key` which conveys the fact that it is the very first KEM public key

Reply:
OK, sure, whatever :)